doc/source/_static/gaffer.css

```diff
@@ -0,0 +1,65 @@
+/* ===================================
+CSS overrides for Gaffer documentation
+==================================== */
+
+/* Match regular nav background to Gaffer brand colors */
+.wy-side-nav-search {
+    background-color: #343131;
+}
+
+/* Match mobile nav background to Gaffer brand colors */
+.wy-nav-top {
+    background-color: #b92227;
+}
+
+/* Increase nav version color visibility */
+.wy-side-nav-search > div.version {
+    color: inherit;
+}
+
+/* Add "Gaffer" in front of nav version (this sidesteps changing the Sphinx template) */
+.wy-side-nav-search > div.version::before {
+    content: "Gaffer ";
+}
+
+/* Make nav search simpler and a little duller when not focused */
+.wy-side-nav-search input[type="text"] {
+    border-width: 2px;
+    border-color: transparent;
+    background-color: lightgrey;
+    background-clip:  padding-box;
+    transition: all 0.3s linear;
+}
+
+/* Transition search bar to white bg with Gaffer red border on focus */
+.wy-side-nav-search input[type="text"]:focus {
+    border-color: #b92227;
+    background-color: white;
+}
+
+/* Make nav button color red on click */
+.wy-menu-vertical a:active {
+    background-color:#b92227;
+}
+
+/* Add icon after external links */
+a[href^="http"]:after {
+    content: " " url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAVklEQVR4Xn3PgQkAMQhDUXfqTu7kTtkpd5RA8AInfArtQ2iRXFWT2QedAfttj2FsPIOE1eCOlEuoWWjgzYaB/IkeGOrxXhqB+uA9Bfcm0lAZuh+YIeAD+cAqSz4kCMUAAAAASUVORK5CYII=);
```

Comment:
I'm not sure how I feel about this one. None of the websites we're linking to use an icon for their links... is this considered standard?

Reply:
It adds the standard external link icon (<img src="https://upload.wikimedia.org/wikipedia/commons/2/25/External.svg" height="10">) to indicate to the user they're being taken to an outside URL. It's a courtesy. We won't be using them often.

Reply:
This will differ from the main site. They're relevant and useful for most knowledge bases and docs. Websites don't use them very often, because there's somewhat of an understanding by the user that they're on the "web," and most body text links are external references (and nav bars are much more heavily used). In docs it's the opposite: most body text links are to other pages in the same "document" (the KB/documentation is considered one unit), and suddenly opening an external link can be jarring and disruptive.

A more complex way of accomplishing this is to use automatic hover-text, but I wanted to keep it simple for now.

Reply:
Ok, fair enough. Thanks for the explanation.